.github/pull_request_template.md

```diff
@@ -6,17 +6,17 @@
 
 ## Pull request checklist
 
-Check the boxes for each DoD items that has been completed:
-
-- **Frontend**
-    - [ ] Changes have been tested
-- **Backend**
-    - [ ] Changes have been tested
-    - [ ] Automatic tests has been added
-    - [ ] OpenAPI definitions have been updated
-    - [ ] Database migrations will work in test environment
-    - [ ] Oracle migration has been updated
-    - [ ] initial.json has been updated to work with migrations
+Check the boxes for each DoD item that has been completed:
+
+  - [ ] Changes have been tested
+  - [ ] Automatic tests have been added
+  - [ ] Test instructions have been written for the customer in the appropriate ticket
+  - [ ] Terminology page in confluence has been updated
+  - [ ] Tooltips have been added in the frontend for all new fields
+  - [ ] OpenAPI definitions have been updated
+  - [ ] Database migrations will work in the DEV & TEST environments
+  - [ ] Oracle migration has been updated
+  - [ ] initial.json has been updated to work with migrations
```

Comment:
Otherwise fine, but I would keep some categorisation on the items:

```
- **Testing**
  - [ ] Changes have been tested
  - [ ] Automatic tests have been added
- **Database**
  - [ ] Database migrations will work in the DEV & TEST environments
  - [ ] initial.json has been updated to work with migrations
  - [ ] Oracle migration has been updated
- **Documentation**
  - [ ] Tooltips have been added in the frontend for all new fields
  - [ ] OpenAPI definitions have been updated
  - [ ] Test instructions have been written for the customer in the appropriate ticket in Jira
  - [ ] Terminology page in Confluence has been updated
```

Reply:
Done